src/Tribe/Generator/Event.php

```diff
@@ -120,21 +121,26 @@ public function create( $quantity = 1, array $args = [], callable $tick = null )
 	 * @param string $recurring_type
 	 * @param string $event_category
 	 * @param string $event_tag
+	 * @param null|int $content_length
+	 *
 	 * @since 1.0.0
 	 * @since 1.0.5 Added Custom Event Category and Tag functionality
+	 * @since TBD Adding content length param.
+	 *
 	 * @return string[]
 	 */
 	public function random_event_data(
 		$from_date, $to_date, $is_featured, $is_virtual,
 		$is_recurring, $recurring_type,
-		$event_category, $event_tag
+		$event_category, $event_tag,
+		$content_length
```

Comment:
The `$content_length` parameter might be `null`: the method should allow for it.

Reply:
Not sure what you mean by that - it does allow for it...?

Reply:
Sorry, I was not very precise.
[Here](https://github.com/mt-support/tribe-ext-test-data-generator/pull/31/files#diff-2755a9c17280041a2cc7e87f0ed0a63e671863372124cbe5ef172cf929084da1R54) `$content_length` is read from the input arguments, or set to `null`.
[Here](https://github.com/mt-support/tribe-ext-test-data-generator/pull/31/files#diff-2755a9c17280041a2cc7e87f0ed0a63e671863372124cbe5ef172cf929084da1R62) that possibly `null` value is passed to the `random_event_data` method.
The `random_event_data` method is not accepting a `null` value, though.